cfgmgr/buffermgr.cpp

```diff
@@ -114,6 +115,65 @@ string BufferMgr::getPgPoolMode()
     return "";
 }
 
+vector<string> BufferMgr::combinePGs(const vector<string> &pgs)
+{
```

Comment:
We have an out-of-box method doing the same thing [set<string> Orch::generateIdListFromMap(unsigned long idsMap, sai_uint32_t maxId)](https://github.com/Azure/sonic-swss/blob/master/orchagent/orch.cpp#L735)
You may want to leverage it.

Reply:
Great. I update code to use the existing one. Thanks